src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs

```diff
@@ -543,6 +543,7 @@ private CodeGenOutput GenerateFastifyPluginOutput(ServiceInfo service)
 			var capModuleName = CodeGenUtility.Capitalize(moduleName);
 			var camelCaseModuleName = CodeGenUtility.ToCamelCase(moduleName);
 			var pluginFileName = CodeGenUtility.Uncapitalize(moduleName) + "Plugin" + (TypeScript ? ".ts" : ".js");
+			var customTypes = new HashSet<string>(service.Dtos.Select(x => x.Name).Concat(service.Enums.Select(x => x.Name)));
```

Comment:
I think when I add .NET 8 support, I'll drop .NET Standard 2.0 from the code gen libraries so they can use modern constructs like `ToHashSet()`.

Reply:
Sounds good!